signalr/signalr.d.ts

```diff
@@ -1,113 +1,344 @@
-// Type definitions for SignalR 1.0
+// Type definitions for SignalR 2.2.0
 // Project: http://www.asp.net/signalr
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>, T. Michael Keesey <https://github.com/keesey/>
```

Comment:
It is a new version written from scratch. Does he have to keep original authors, even though the definitions were written looking at the source directly?